src/test/scala/uk.gov.nationalarchives.fileformat/FileUtilsTest.scala

```diff
@@ -16,25 +16,29 @@ import org.scalatest.matchers.should.Matchers._
 import sangria.ast.Document
 import software.amazon.awssdk.services.s3.S3Client
 import software.amazon.awssdk.services.s3.model.{GetObjectRequest, GetObjectResponse}
-import sttp.client.{HttpError, Response}
+import sttp.client.asynchttpclient.WebSocketHandler
+import sttp.client.asynchttpclient.future.AsyncHttpClientFutureBackend
```

Comment:
Think sttp.client.asynchttpclient.future.AsyncHttpClientFutureBackend is an unnecessary import